File: src/main/java/com/example/store/dto/BillRequest.java
package com.example.store.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class BillRequest {
    private String productName;
    private Integer price;
    private Integer qty;
    private String type;
}

File: src/main/kotlin/com/example/store/dto/BillRequest.kt
package com.example.store.dto

class BillRequest {
    private var productName: String? = null
    private var price: Int? = null
    private var qty: Int? = null
    private var type: String? = null

    fun getPrice(): Int? {
        return price
    }

    fun setPrice(price: Int) {
        this.price  = price
    }

    fun getQty(): Int? {
        return qty;
    }

    fun setQty(qty: Int) {
        this.qty  = qty
    }

    fun getType(): String? {
        return type
    }

    fun setType(type: String) {
        this.type  = type
    }
}